src/xdp/program.c

```diff
@@ -1568,7 +1567,6 @@ XdpIrpCreateProgram(
     return Status;
 }
 
-static
```

Comment:
this should go back to being static if we create a single XdpCloseHandle routine.

Reply:
@nigriMSFT 
Why did you remove static?

Reply:
I removed static so I could call the fn from a different module (apikernel). The suggestion in another comment about single XdpCloseHandle can allow us to revert the static removal